app/controllers/patients/edit_controller.rb

```diff
@@ -14,9 +14,7 @@ def update_nhs_number
 
     @existing_patient =
       if nhs_number.present?
-        policy_scope(Patient).includes(parent_relationships: :parent).find_by(
-          nhs_number:
-        )
+        Patient.includes(parent_relationships: :parent).find_by(nhs_number:)
```

Comment:
Would it be worth limiting the scope of patients here to be either those connected to the user's organisation OR with no organisation, versus what we have here which could move patients from other organisations into the user's?